zssh/zscp/main.go

```diff
@@ -1,5 +1,247 @@
 package main
 
+import (
+	"fmt"
+	"github.com/openziti/sdk-golang/ziti"
+	"github.com/openziti/sdk-golang/ziti/config"
+	"github.com/pkg/sftp"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"io/fs"
+	"log"
+	"os"
+	"os/user"
+	"path/filepath"
+	"runtime"
+	"strings"
+	"zssh/zsshlib"
+)
+
+const ExpectedServiceAndExeName = "zssh"
+
+var (
+	ZConfig     string
+	SshKeyPath  string
+	debug       bool
+	recursive   bool
+	serviceName string
+
+	rootCmd = &cobra.Command{
+		Use: "Remote to Local: zscp <remoteUsername>@<targetIdentity>:[Remote Path] [Local Path]\n" +
+			"Local to Remote: zscp [Local Path] <remoteUsername>@<targetIdentity>:[Remote Path]",
+		Short: "Z(iti)scp, Carb-loaded ssh performs faster and stronger than ssh",
+		Long:  "Z(iti)scp is a version of ssh that utilizes a ziti network to provide a faster and more secure remote connection. A ziti connection must be established before use",
+		Args:  cobra.ExactValidArgs(2),
+		Run: func(cmd *cobra.Command, args []string) {
+			if SshKeyPath == "" {
+				userHome, err := os.UserHomeDir()
+				if err != nil {
+					logrus.Fatal(err)
+				}
+				SshKeyPath = filepath.Join(userHome, ".ssh", "id_rsa")
+			}
+			if debug {
+				logrus.Infof("    sshKeyPath set to: %s", SshKeyPath)
+			}
+
+			if ZConfig == "" {
+				userHome, err := os.UserHomeDir()
+				if err != nil {
+					logrus.Fatal(err)
+				}
+				ZConfig = filepath.Join(userHome, ".ziti", fmt.Sprintf("%s.json", ExpectedServiceAndExeName))
+			}
+			if debug {
+				logrus.Infof("       ZConfig set to: %s", ZConfig)
+			}
+
+			var username string
+			var targetIdentity string
+			var remoteFilePath string
+			var localFilePath string
+			var isCopyToRemote bool
+
+			if strings.ContainsAny(args[0], ":") {
+				remoteFilePath = args[0]
+				localFilePath = args[1]
+				isCopyToRemote = false
+
+			} else if strings.ContainsAny(args[1], ":") {
+				remoteFilePath = args[1]
+				localFilePath = args[0]
+				isCopyToRemote = true
+			} else {
+				logrus.Fatal("cannot determine remote file PATH use \":\" for remote path")
```

Comment:
for something like this you can use a string literal and not need to esacpe the quote character:

`logrus.Fatal(`cannot determine remote file PATH use ":" for remote path`)`